paymentsheet/src/main/java/com/stripe/android/paymentsheet/DeferredIntentValidator.kt

```diff
@@ -75,7 +75,7 @@ internal object DeferredIntentValidator {
         intentConfiguration: PaymentSheet.IntentConfiguration,
     ): DeferredIntentParams {
         val initializationMode = PaymentSheet.InitializationMode.DeferredIntent(intentConfiguration)
-        val params = initializationMode.toElementsSessionParams(externalPaymentMethods = null)
+        val params = initializationMode.toElementsSessionParams(customer = null, externalPaymentMethods = null)
```

Comment:
I'm worried we will forget this spot. Can you make sure it's in a task to update?

Reply:
I looked into this code and it looks like we only use the function here to create `DeferredIntentParams`. I can make a task to migrate off using this function for this case so it's not confusing.